src/tests/blueprint/t_blueprint_mesh_verify.cpp

```diff
@@ -750,6 +850,35 @@ TEST(conduit_blueprint_mesh_verify, topology_unstructured)
         n["elements"]["c"]["offsets"].set(DataType::int32(5));
         CHECK_MESH(verify_unstructured_topology,n,info,true);
 
+        n["elements"]["d"]["shape"].set("polygonal");
+        n["elements"]["d"]["connectivity"].set(DataType::int32(6));
+        CHECK_MESH(verify_unstructured_topology,n,info,false);
+        n["elements"]["d"]["offsets"].set(DataType::int32(3));
+        CHECK_MESH(verify_unstructured_topology,n,info,false);
+        n["elements"]["d"]["sizes"].set(DataType::int32(2));
+        CHECK_MESH(verify_unstructured_topology,n,info,false);
+        n["elements"]["d"]["sizes"].set(DataType::int32(3));
+        CHECK_MESH(verify_unstructured_topology,n,info,true);
+
+        n["elements"]["e"]["shape"].set("polyhedral");
+        n["elements"]["e"]["connectivity"].set(DataType::int32(6));
+        CHECK_MESH(verify_unstructured_topology,n,info,false);
+        n["elements"]["e"]["offsets"].set(DataType::int32(3));
+        CHECK_MESH(verify_unstructured_topology,n,info,false);
+        n["elements"]["e"]["sizes"].set(DataType::int32(2));
+        CHECK_MESH(verify_unstructured_topology,n,info,false);
+        n["elements"]["e"]["sizes"].set(DataType::int32(3));
+        CHECK_MESH(verify_unstructured_topology,n,info,false);
+
+        n["subelements"]["e"]["shape"].set("polygonal");
+        CHECK_MESH(verify_unstructured_topology,n,info,false);
+        n["subelements"]["e"]["connectivity"].set(DataType::int32(6));
+        CHECK_MESH(verify_unstructured_topology,n,info,false);
+        n["subelements"]["e"]["offsets"].set(DataType::int32(3));
+        CHECK_MESH(verify_unstructured_topology,n,info,false);
+        n["subelements"]["e"]["sizes"].set(DataType::int32(3));
+        CHECK_MESH(verify_unstructured_topology,n,info,true);
+
```

Comment:
Thanks for doing such thorough testing of the new schemas and for filling the gaps in mixed shape topology testing!